services/essentials.yaml

```diff
@@ -87,6 +129,15 @@ spec:
     - groupId: org.jenkins-ci.plugins
       artifactId: jira
       version: 3.0.2
+    - groupId: org.jenkins-ci.ui
+      artifactId: ace-editor
```

Comment:
Hmm, we sort the `status` when we write it out; should we not also sort the `spec`?

(My advice remains to get rid of the separate `spec` and `status` sections.)